packages/zero-pg/src/query.pg-test.ts

```diff
@@ -0,0 +1,36 @@
+import {beforeEach, describe, expect, test} from 'vitest';
+import type {SchemaQuery} from '../../zql/src/mutate/custom.ts';
+import type {PostgresDB} from '../../zero-cache/src/types/pg.ts';
+import {schema, schemaSql, seedDataSql} from './test/schema.ts';
+import {testDBs} from '../../zero-cache/src/test/db.ts';
+import {makeSchemaQuery} from './query.ts';
+import {Transaction} from './test/util.ts';
+import type {DBTransaction} from './db.ts';
+
+describe('makeSchemaQuery', () => {
+  let pg: PostgresDB;
+  let queryProvider: (tx: DBTransaction<unknown>) => SchemaQuery<typeof schema>;
+
+  beforeEach(async () => {
+    pg = await testDBs.create('makeSchemaQuery-test');
+    await pg.unsafe(schemaSql);
+    await pg.unsafe(seedDataSql);
+
+    queryProvider = makeSchemaQuery(schema);
+  });
+
+  test('select', async () => {
```

Comment:
this is just to test the plumbing. Tests of query forms (related/where/exists/etc.) are in `z2s/test/chinook`